lib/matplotlib/dviread.py

```diff
@@ -194,7 +180,7 @@ class Dvi(object):
     >>>         print(''.join(unichr(t.glyph) for t in page.text))
     """
     # dispatch table
-    _dtable = [None for _ in xrange(256)]
+    _dtable = [None for _ in range(256)]
```

Comment:
Slightly out of scope, but this would be simpler and faster as `[None] * 256`, wouldn't it?